local-cluster/src/local_cluster.rs

```diff
@@ -63,6 +69,52 @@ use {
     },
 };
 
+pub fn build_tpu_quic_client(
+    cluster: &LocalCluster,
+) -> Result<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>> {
+    build_tpu_client(cluster, |rpc_url| Arc::new(RpcClient::new(rpc_url)))
+}
+
+pub fn build_tpu_quic_client_with_commitment(
+    cluster: &LocalCluster,
+    commitment_config: CommitmentConfig,
+) -> Result<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>> {
+    build_tpu_client(cluster, |rpc_url| {
+        Arc::new(RpcClient::new_with_commitment(rpc_url, commitment_config))
+    })
+}
```

Comment:
What do you think about extending `trait Cluster` with these methods? (And maybe down the road deprecating `Cluster::get_validator_client()`, which returns a `ThinClient`...)

Reply:
haha in the process! working on another PR for extending `trait Cluster`

Reply:
Okay, great. Let's not bother adding these public, stand-alone methods, then.